### PR TITLE
extra conditions for slow request alarms

### PR DESCRIPTION
**Pull request by agalamatis**
trello: https://trello.com/c/Lgu5r2Rr/773-exclude-slow-requests-over-a-certain-size-from-triggering-alarms

extra conditions are:
- request begins with POST
- size larger than 500k

### CODE REVIEW

File: terraform/modules/logging/log-metric-filters/main.tf [hunk 1]
```diff
@@ -285,7 +285,7 @@ resource "aws_cloudwatch_log_metric_filter" "request_time_bucket_router_8" {
 
 resource "aws_cloudwatch_log_metric_filter" "request_time_bucket_router_9" {
   name           = "${var.environment}-router-request-times-9"
-  pattern        = "{$$.requestTime >= 10 && $$.request != \"*/_status?ignore-dependencies *\"}"
+  pattern        = "{$$.requestTime >= 10 && $$.request != \"*/_status?ignore-dependencies *\" && $$.request != \"POST*\" && $$.size < 500000}"
```
Comment (katstevens) [inline terraform/modules/logging/log-metric-filters/main.tf]: Will this include a slow POST with a small request size?

Comment (agalamatis) [reply terraform/modules/logging/log-metric-filters/main.tf]: all conditions operated by &&, so all four must be true to send the alarm

Comment (katstevens) [reply terraform/modules/logging/log-metric-filters/main.tf]: I think we probably want to know about small slow POSTs. I'd be happy to remove the `!= POST` condition.

Comment (agalamatis) [reply terraform/modules/logging/log-metric-filters/main.tf]: I can also separate the two conditions with an || and put them into brackets.

**Pull request merged**